### PR TITLE
Telugu Cheat Sheet: Removed "english to telugu" and "basic telugu" aliases

### PR DESCRIPTION
**Pull request by ethanchewy**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [X] Improvement
    - [X] Bug fix: **`{Telugu Cheat Sheet}: Fix {2967}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Removed "english to telugu" and "basic telugu"

###### Which issues (if any) does this fix?

Fixes (partially) #2967 

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/telugu_cheat_sheet

[Maintainer](https://github.com/roxxup): @roxxup

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by GuiltyDolphin** [comment]
@ethanchewy Could you please fill out the [pull request template](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/.github/PULL_REQUEST_TEMPLATE.md).

**Comment by ethanchewy** [comment]
@GuiltyDolphin Oops! My bad. Below is the template filled out.
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ X] Improvement
    - [ X] Bug fix: **`{Telugu Cheat Sheet}: Fix {2967}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Removed "english to telugu" and "basic telugu"

###### Which issues (if any) does this fix?

Fixes #2967 

###### People to notify (@mention interested parties)


---

Instant Answer Page: https://duck.co/ia/view/telugu_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @roxxup

File: share/goodie/cheat_sheets/json/language/telugu.json [hunk 1]
```diff
@@ -6,9 +6,7 @@
         "sourceName":"Wikitravel",
         "sourceUrl":"http://wikitravel.org/en/Telugu_phrasebook"
     },
-    "aliases":[  
-        "english to telugu",
-        "basic telugu"
+    "aliases":[
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/language/telugu.json]: Lets remove the aliases array altogether.

File: share/goodie/cheat_sheets/json/language/telugu.json [hunk 2]
```diff
@@ -6,10 +6,7 @@
         "sourceName":"Wikitravel",
         "sourceUrl":"http://wikitravel.org/en/Telugu_phrasebook"
     },
-    "aliases":[  
-        "english to telugu",
-        "basic telugu"
-    ],
+    "aliases":,
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/language/telugu.json]: We need to remove the key entirely.

**Comment by GuiltyDolphin** [comment]
@ethanchewy Thanks!

**Comment by ethanchewy** [comment]
@GuiltyDolphin No Problem. Sorry for the delays!

**Comment by rahulroxx** [comment]
Thanks for noticing :+1:

**Pull request merged**